src/binder/bind_node_visitor.cpp

```diff
@@ -171,9 +171,11 @@ void BindNodeVisitor::Visit(parser::LimitDescription *) {}
 void BindNodeVisitor::Visit(parser::CopyStatement *) {}
 void BindNodeVisitor::Visit(parser::CreateFunctionStatement *) {}
 void BindNodeVisitor::Visit(parser::CreateStatement *node) {
+  LOG_INFO("aaron: CreateStatement");
```

Comment:
Remove.

Reply:
I'm wondering if you think it's a good idea to wrap up the big switch using a function.